bin/build_utils

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
+
+set -euo pipefail
+
+####
+# Functions to generate version numbers for this project
+####
+
+readonly VERSION_GO_FILE="pkg/trace/version.go"
+
+function git_version() {
+  grep -v '^//' "${VERSION_GO_FILE}" | grep 'const Version =' | awk -F'= ' '{print $2}' | tr -d '"'
+}
+
+function git_commit_short() {
+  git rev-parse --short HEAD
```

Comment:
The length returned of `--short` here is based on number of commits for it to be unique.  Rather than hope it stays the same, the length should be specified with `--short=8`.